libraries/AP_Camera/AP_Camera_Backend.h

```diff
@@ -125,6 +128,13 @@ class AP_Camera_Backend
         uint32_t feedback_trigger_logged_count; // ID sequence number
     } camera_feedback;
 
+    // Picture settings
+    struct {
+        bool enabled;               // true if taking multiple pictures is enabled
```

Comment:
We could get rid of the enabled variable completely.. num_remaining basically tells us if we've got any pictures to take or not.